src/main/java/app/view/FileMenuBar.java

```diff
@@ -30,8 +44,41 @@ public FileMenuBar(App app)
         });
         file.getItems().add(newFile);
 
+
+        /* Open file menu*/
+        /*
+        Menu openFile = new Menu("Open");
+        for(File f: filterErrorFiles(getResourceFolderFiles("")))
+        {
+            MenuItem fileOption = new MenuItem(f.getName());
+            openFile.getItems().add(fileOption);
+            fileOption.setOnAction(e -> app.gotoSimulation(f.getName()));
+        }
+        file.getItems().add(openFile);
+        */
```

Comment:
This code should either be live, or be removed.  Commented out code in `Master` should be avoided at all costs!